src/Cake.Common/Text/TextTransformation.cs

```diff
@@ -49,9 +49,10 @@ public sealed class TextTransformation<TTemplate>
         /// <summary>
         /// Saves the text transformation to a file.
         /// </summary>
-        /// <param name="path">The <see cref="FilePath"/> to save the test transformation to.</param>
+        /// <param name="path">The <see cref="FilePath"/> to save the text transformation to.</param>
+        /// <param name="encoding">The text encoding. Defaults to UTF8 without BOM.</param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times", Justification = "Stream writer leaves stream open.")]
-        public void Save(FilePath path)
+        public void Save(FilePath path, Encoding encoding = null)
```

Comment:
To not break ABI add an `public void Save(FilePath path)` overload calling the new one instead of default parameter.